src/main/java/main.java

```diff
@@ -0,0 +1,137 @@
+import delete_post.drivers.DeletePostDataAccess;
+import delete_post.interface_adapters.DeletePostController;
+import delete_post.interface_adapters.DeletePostPresenter;
+import delete_post.interface_adapters.DeletePostViewModel;
+import delete_post.use_case.DeletePostInteractor;
+import filter_post.drivers.FilterPostDataAccess;
+import filter_post.interface_adapters.FilterPostController;
+import filter_post.interface_adapters.FilterPostPresenter;
+import filter_post.interface_adapters.FilterPostViewModel;
+import filter_post.ui.FilterPostBarView;
+import filter_post.use_case.FilterPostInteractor;
+import make_post.drivers.MakePostDatabaseAccess;
+import make_post.interface_adapters.MakePostController;
+import make_post.interface_adapters.MakePostPresenter;
+import make_post.interface_adapters.MakePostViewModel;
+import make_post.ui.MakePostView;
+import make_post.use_case.MakePostInteractor;
+import make_post.use_case.PostFactory;
+import sign_up.drivers.SignUpDatabaseAccess;
+import sign_up.interface_adapters.SignUpController;
+import sign_up.interface_adapters.SignUpPresenter;
+import sign_up.interface_adapters.SignUpViewModel;
+import sign_up.ui.MasterLandingView;
+import sign_up.ui.SignUpView;
+import sign_up.use_case.SignUpInteractor;
+import sign_up.use_case.UserFactory;
+import view_post.drivers.ViewPostGateway;
+import view_post.interface_adapters.ViewPostController;
+import view_post.interface_adapters.ViewPostPresenter;
+import view_post.interface_adapters.ViewPostViewModel;
+import view_post.ui.MainFrame;
+import view_post.ui.PostListView;
+import view_post.ui.ViewPostView;
+import view_post.use_case.ViewPostInteractor;
+
+/*
+* The main class for the file, containing the main method which runs the program
+* */
+public class main {
```

Comment:
Improper naming convention for classes. Needs to be uppercase but refer to general review comments for suggested changes.